### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VR56**

### CODE REVIEW

File: Assignment_1_Problem_3.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            return value1 - value2;
+            break;
+
+        case '*':
+            return value1 * value2;
+            break;
+
+        case '/':
+            return value1 / value2;
+            break;
```
Comment (vrallabhandi) [inline Assignment_1_Problem_3.js]: Its good to have a default case as a practice when using `switch-cases`. you can return null here

File: Assignment_1_Problem_4.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+// Solution
+
+function rentalCarCost(d) {
+    if (d >= 3 && d < 7)
+        return (d * 40) - 20;
+    else if (d >= 7)
+        return (d * 40) - 50;
+    else if (d < 3)
+        return d * 40;
```
Comment (vrallabhandi) [inline Assignment_1_Problem_4.js]: `d * 40` can be moved to a variable. rename the variable `d` to have a readable name. Avoid having too many return statements in the code.

**Comment by VR56** [comment]
Hi Siva,
I updated my code with the changes asked

**Comment by VR56** [comment]
Hi Siva,
I pushed the Assignment2's files in the same branch as the last files of assignment are not yet merged
please Review and comment the necessary changes required.
Thank You

**Pull request closed without merging**